dcs/mapping.py

```diff
@@ -97,6 +97,11 @@ class Vector2:
     x: float
     y: float
 
+@dataclass
+class WWIIFollowBigFormationOrder:
+    x: int # distance
+    y: int # elevation
+    z: int # interval
```

Comment:
What is this really? Distance between individual aircraft? Distance between subgroups? I don't know how big formation works so it's not obvious to me. Whichever it is, use a name that makes it clear.